packages/okta-react-native/LICENSE

```diff
@@ -0,0 +1,187 @@
+The Okta software accompanied by this notice is provided pursuant to the
+following terms:
+
+Copyright © 2015-2018, Okta, Inc.
```

Comment:
If this is the first year that we've written the code in this library, then we should have just 2018 for now.